gatsby-node.mjs

```diff
@@ -12,6 +12,13 @@ import redirects from './redirects.json' assert {
 
 export const onCreateWebpackConfig = ({ actions, stage, plugins }) => {
   actions.setWebpackConfig({
+    devtool: "cheap-module-source-map",
+    parallelism: 4,
```

Comment:
This looks really low

Reply:
The default value is 100.

Reply:
Well, just made some tests and this option does not seems to really change something, I'll remove it